Frontend/ui-customer/src/Screens/TrackRide/FollowRideScreen/ScreenData.purs

```diff
@@ -172,6 +173,7 @@ mockDriverInfo =
   , driverName: "Test Driver"
   , eta: Nothing
   , vehicleDetails: ""
+  , currentSearchResultType: ST.ESTIMATES
```

Comment:
Can we handle this use case using FareProduct type itself , instead of creating a separate variable for differentiating between QUOTES and ESTIMATES.